test/server/controllers/offers.test.ts

```diff
@@ -60,27 +60,42 @@ describe('Offer Tests', async () => {
         })
     })
 
-    describe('#POST /api/offers/createOffer', async () => {
+    describe('#POST /api/offers/', async () => {
         context('No offer created', async () => {
-            let createOfferClone, user
-            beforeEach(async () =>  {
-                const user = await factory.create('user', createUser) 
+            let createOfferClone, user, createCategoryClone, categoryId
+            beforeEach(async () => {
+                user = await factory.create('user', createUser)
                 createOfferClone = _.cloneDeep(createOffer)
                 createOfferClone.userId = user.id
+                createCategoryClone = _.cloneDeep(createCategory)
+                createCategoryClone.id = '319497a7-73fe-4e1c-b224-8f71d0ee0bb8'
+                const cat = await factory.create('category', createCategoryClone)
+                categoryId = cat.id
             })
             it('Should create the new offer', async () => {
                 const token = (await logInUser(user)).accessToken
                 const res = await request.post('/api/offers/').set('Authorization', `bearer ${token}`)
-                .send(createOfferClone).expect(200)
-                //logger.info('Response => ' + JSON.stringify(res.body))
+                    .send(createOfferClone).expect(200)
```

Comment:
We should expect something to happen. We should validate the created offer and verify that does match the provided data